common/locales/id.json

```diff
@@ -95,13 +95,17 @@
     "binds": {
         "adjustOffset": "Sesuaikan offset takarir ±100 ms",
         "adjustOffsetToNextSubtitle": "Sesuaikan offset takarir agar takarir berikutnya berada di waktu saat ini",
+        "increaseSubtitlePositionOffset": "Tambah Offset Posisi Takarir",
```

Comment:
Some key names changed. You'll need to remove the old ones and add the new ones:
- `increaseSubtitlePositionOffset` -> `moveBottomSubtitlesUp`
- `decreaseSubtitlePositionOffset` -> `moveBottomSubtitlesDown`
- `increaseTopSubtitlePositionOffset` -> `moveTopSubtitlesUp`
- `decreaseTopSubtitlePositionOffset` -> `moveTopSubtitlesDown`